extensions/eclipse-che-theia-testing-service-ext/src/browser/languages-test-api.ts

```diff
@@ -0,0 +1,186 @@
+/********************************************************************************
+ * Copyright (C) 2020 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
```

Comment:
Since we own the code 100%, we can make it straight EPL 2.0, no? Please search through the code and fix all locations.